config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  namespace :v1, defaults: { format: 'json' } do
+    get 'message', to: 'messages#index'
+  end
+  get '*page', to: 'statics#index', constraints: lambda { |req|
+    !req.xhr? && req.format.html?
+  }
+  root 'statics#index'
```

Comment:
- [ ] Kindly fix the root page view , I'm getting this error
![image](https://user-images.githubusercontent.com/67957533/146187174-fd010901-b806-4a6d-84b3-88f8be939d36.png)